src/views/Home/CDKTypeTabs.tsx

```diff
@@ -30,6 +30,7 @@ interface PackageTabProps {
 const PackageTab: FunctionComponent<PackageTabProps> = ({ cdkType, data }) => {
   return (
     <Tab
+      _selected={{ borderBottom: "4px solid #AED5FF" }}
```

Comment:
```suggestion
      _selected={{ borderBottomColor: "white" }}
```

Reply:
The design is to use a different color. Should I use white?

Reply:
My bad - I wasn't aware there were design changes outside of this new section - disregard

Reply:
Cool, so leaving as is to align with design.